src/admin/index.js

```diff
@@ -69,6 +69,16 @@ module.exports = ({
     instrumentationEmitter.emit(DISCONNECT)
   }
 
+  /**
+   * @return {Promise}
+   */
+  const listTopics = async () => {
+    await cluster.refreshMetadata()
```

Comment:
This should not be necessary. `cluster.metadata` will refresh if it needs to. `refreshMetadata` will also just refresh metadata for topics it knows about, so it wouldn't be able to discover new topics anyway.